### PR TITLE
Add clarifying instructions to Google Docs import/export README

### PR DESCRIPTION
**Pull request by matthewsilver**

### CODE REVIEW

**Comment by liquidsteves** [comment]
You need to pull in the current version of master. Right now, you are missing a PR I merged in a couple weeks ago adding docs on Wordpress to this doc.

File: import_export/README.md [hunk 1]
```diff
@@ -2,18 +2,29 @@
 
 ## Overview
 
-This folder includes the code for importing data from and exporting data to various external data sources where original documents and translated content is hosted. Currently, Google documents and Wordpress sites are the two data sources integrated into the ELSA continuous localization pipeline.
```
Comment (liquidsteves) [inline import_export/README.md:5]: Why delete this sentence?

File: import_export/README.md [hunk 2]
```diff
@@ -2,18 +2,29 @@
 
 ## Overview
 
-This folder includes the code for importing data from and exporting data to various external data sources where original documents and translated content is hosted. Currently, Google documents and Wordpress sites are the two data sources integrated into the ELSA continuous localization pipeline.
+This folder includes the code for importing data from and exporting data to various external data sources where original documents and translated content is hosted.
 
-## Google Docs
+## Setup
```
Comment (liquidsteves) [inline import_export/README.md:7]: Keep "Google Docs"

File: import_export/README.md [hunk 3]
```diff
@@ -26,49 +37,22 @@ Translations
...
-Click "Send Demo". This should cause it to ping your flask endpoint.
-
-From the main menu on the left hand side of the page, go to Posts -> All Posts and click "Edit" under "Hello World".
-Make a change to the text of the page and click "Update".
-This should cause it to ping your flask endpoint.
-
-### Exporting to Wordpress
-
-TODO
-
```
Comment (liquidsteves) [inline import_export/README.md:74]: Undo this deletion, keep it at the bottom

**Review by liquidsteves — COMMENTED** [review]
Thanks for adding this documentation! It looks super useful.

Glad we are getting these docs in good shape for more new folks to use. ELSA ftw!

File: import_export/README.md (hunk 1 above)
Comment (matthewsilver) [reply import_export/README.md:5]: I was removing the wordpress stuff from this readme but forgot this module includes both google docs and wordpress so will add back

**Pull request merged**